tasks/main.yml

```diff
@@ -129,3 +125,11 @@
     state: "{{ item.state | default(present) }}"
     reload: "{{ item.reload | default(False) }}"
   with_items: "{{ selinux_logins }}"
+
+- include_tasks: selinux_load_module.yml
+  when: selinux_modules is defined
+  loop: "{{ selinux_modules | flatten(levels=1) }}"
```

Comment:
To "pass variables" to the included tasks file, use `vars:` here:
```yaml
- include_tasks: selinux_load_module.yml
  when: selinux_modules is defined
  loop: "{{ selinux_modules | flatten(levels=1) }}"
  vars:
    name: "{{ item.name | default('') }}"
    state: "{{ item.state | default('enabled') }}"
    priority: "{{ item.priority | default('400') }}"
    store: "{{ selinux_policy | default(ansible_selinux.type, true) }}"
```
that way the variables will be local, instead of global.